src/rtctools_heat_network/heat_mixin.py

```diff
@@ -2770,7 +2770,7 @@ def __ates_heat_to_discharge_path_constraints(self, ensemble_member):
             # the constraints with a boolean. Note that `discharge` and `heat_hot`
             # are guaranteed to have the same sign.
             flow_dir_var = self.__pipe_to_flow_direct_map[hot_pipe]
-            is_ates_charging = 1 - self.state(flow_dir_var)
+            is_ates_charging = self.state(flow_dir_var)
 
             big_m = self.__get_abs_max_bounds(
                 *self.merge_bounds(bounds[f"{a}.HeatIn.Heat"], bounds[f"{a}.HeatOut.Heat"])
```

Comment:
constraints on ates need to be changed because now during discharging (Q and HEat_ates both negative, but abs(heat_ates) should be larger than abs(cp rho dT Q)

Reply:
fixed